src/Consumer.cc

```diff
@@ -45,8 +46,48 @@ void Consumer::Init(Napi::Env env, Napi::Object exports) {
   constructor.SuppressDestruct();
 }
 
+struct MessageListenerProxyData {
+  Consumer *consumer;
+  pulsar_message_t *cMessage;
```

Comment:
`cMessage` should be declared before `consumer`. Otherwise, you will get warnings when compiling.
```
../src/Consumer.cc:51:21: warning: ‘MessageListenerProxyData::cMessage’ will be initialized after [-Wreorder]
   pulsar_message_t *cMessage;
                     ^
../src/Consumer.cc:50:13: warning:   ‘Consumer* MessageListenerProxyData::consumer’ [-Wreorder]
   Consumer *consumer;
             ^
../src/Consumer.cc:53:3: warning:   when initialized here [-Wreorder]
   MessageListenerProxyData(pulsar_message_t *cMessage, Consumer *consumer)
   ^
```

Reply:
Didn't saw those errors because I worked with `npm run build:debug`, now I see them and fixed.
Thanks.